conbench/runner.py

```diff
@@ -246,11 +264,14 @@ def record(self, result, name, error=None, publish=True, **kwargs):
             output,
         ) = self._init(kwargs)
 
-        if (tags.get("name") is not None) and (name == tags["name"]):
+        if (tags.get("name") is not None) and (name != tags["name"]):
```

Comment:
Thank you! Agreed this was the intent

Reply:
yes, this is the right intent — should we add a test to cover this?